services/vision/mlvision/ml_model.go

```diff
@@ -69,18 +69,31 @@ func registerMLModelVisionService(
 	if err != nil {
 		return nil, err
 	}
+
 	classifierFunc, err := attemptToBuildClassifier(mlm)
 	if err != nil {
-		logger.Infow("error turning turn ml model into a classifier:", "model", params.ModelName, "error", err)
+		logger.Infow("%v", errors.Wrapf(err, "was not able to turn ml model %q into a classifier", params.ModelName))
```

Comment:
The way `Infow` works is like a map: You have an initial message, and then a map of messages, like:
```
log.Infow("here is relevant info",
"timestamp", time.Now(),
"name", thing.Name,
"error", err,
)
```

So in your case, you'll want to write something like
`logger.Infow("was not able to turn ml model into classifier", "ml model", params.ModelName, "error", err)`